reproducible/ubuntu/build.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
```

Comment:
I'd drop the build script here. It doesn't really add anything, and we should be using triggers or running this ourselves.